aikau/src/main/resources/alfresco/documentlibrary/AlfDocumentFilters.js

```diff
@@ -154,11 +153,11 @@ define(["dojo/_base/declare",
       /**
        * @instance
        */
-      onShowMoreClick: function alfresco_documentlibrary_AlfDocumentFilters__onShowMoreClick(evt) {
+      onShowMoreClick: function alfresco_documentlibrary_AlfDocumentFilters__onShowMoreClick() {
```

Comment:
This feels inconsistent. Why remove the unused param here, but annotate others with "unused:false"?